src/bb/image-io/rt_u3v.h

```diff
@@ -484,6 +484,22 @@ class U3V {
         return err_;
     }
 
+    void ValidateUserInput(int32_t num_detected_device, char* dev_id){
```

Comment:
Let's unify the naming convention. In ion-kit, we use `snake_case` for the function and variable name.

Reply:
solved in [0a63a5b](https://github.com/fixstars/ion-kit/pull/324/commits/0a63a5bd39dd9c21526c6edadbc28402d3ee6e5e)